docker-compose.yml

```diff
@@ -48,7 +49,7 @@ services:
     image: "eth_worker:${GIT_HASH}"
     environment:
       DEPLOYMENT_NAME: "DOCKER_TEST"
-      CONTAINER_TYPE: "PRIMARY"
+      CONTAINER_TYPE: "HIGH_PRIORITY_WORKER"
```

Comment:
This is the part that's currently breaking tests, as it's diverting the worker from the default config (no match) to HIGH_PRIORITY.

Suggest changing it back for now and we review worker splits in a seperate PR...